### PR TITLE
sorted logos using SimilarWeb global rank as sorting indicator

### PR DESCRIPTION
**Pull request by manpenaloza**
So far I've implemented basic functionality requested in #439 but without the api request logic i mentioned in #439 (so far no feedback from SimilarWeb on my request for an api key). So did this searching/typing thingy to implement and test the feature organization taking in consideration an eventually upcoming SimilarWeb api implementation. But, data is valid, the component works as it should as far as I see and thus could be used for deployment if you want.

Some notes on the implementation:
- considering handling the global rank data receipt via api, I had to replace the React Fragment implementation with javascript array. Reason: a Fragment doesn't list its children (which would be all the logos), but using the api I'd need the children (logos) in order do to a proper element cloning appending the `similarWebGlobalRank` property.
- tried to modify only what is needed to achieve this, so rendering logic and base idea of a huge component including all logo components kept the same
- properties `key` and `similarWebGlobalRank` were added separately to the company logo component. but those props are highly company related. so compared this implementation it probably would make sense to build up company related datasets/objects which can be used to render company related stuff.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -237,128 +237,150 @@ const FooterContent = styled(Content)`
...
+
+  <CompanyLogo bottom="-0.2rem" height="1.75rem" key="https://www.bloomberg.com" similarWebGlobalRank={507}>
+    <BloombergLogo />
+  </CompanyLogo>,
+
+  <CompanyLogo bottom="0.75rem" height="3rem" key="https://www.blue-tomato.com" similarWebGlobalRank={21322}>
+    <BlueTomatoLogo />
+  </CompanyLogo>,
+
+  <CompanyLogo bottom="-0.2rem" key="http://casperjs.org/" similarWebGlobalRank={1287463}>
```
Comment (mxstbr) [inline pages/index.js]: This is actually `casper.com`, which should have a much higher rank!

Comment (manpenaloza) [reply pages/index.js]: thx, updated it.

**Comment by manpenaloza** [comment]
updated the branch with the `Casper` fix, ready for another review.

File: pages/index.js [hunk 2]
```diff
@@ -237,128 +237,150 @@ const FooterContent = styled(Content)`
...
+  );
 
-      <CompanyLogo bottom="0.7rem" height="2.5rem">
-        <XingLogo />
-      </CompanyLogo>
-    </>
+  return (
+    <UsersWrapper>
+      {similarWebGlobalRankSortedLogos}
+      {similarWebGlobalRankSortedLogos}
```
Comment (mxstbr) [inline pages/index.js]: Is there a reason they are rendered twice?

Comment (kitten) [reply pages/index.js]: That's for the animation but previously the logos were in a component that was rendered twice, so this is correct 👌 there might be a smarter way to make this work though

Comment (manpenaloza) [reply pages/index.js]: jap, saw it was rendered twice before, so I just kept it for now.

**Pull request merged**